lib/src/widget/nav_bar/data/nav_bar_config.dart

```diff
@@ -73,9 +73,9 @@ class NavBarConfig extends Equatable {
       const NavBarConfig._(idIgnored, [], false, NavBarType.ignored, false);
 
   NavBarConfig.backBtn(
-    NavBarItemBackButton btn,
-  )   : id = idBackBtn,
-        items = [btn],
+    NavBarItemBackButton btn, {
+    this.id = idBackBtn,
```

Comment:
🤔 maybe its better to make it `required`? to avoid this problem in the future?
wdyt?